### PR TITLE
AE-171 fix tag push workflows

### PR DESCRIPTION
**Pull request by MkMan**
Since adding a second workflow triggered by tag pushes both tag triggered workflows have been running intermittently. I couldn't find a reason why online but I have a theory that it's having 2 workflows triggered by the same event. 

This PR combines the 2 workflows into 1. I tested it with 3 tag pushes and it ran every time, hopefully it stays that way 🤞 
image

### CODE REVIEW

File: .github/workflows/tag-push.yml [hunk 1]
```diff
@@ -1,12 +1,49 @@
...
+
+      - name: Checkout private tools
+        uses: actions/checkout@v2
+        with:
+          repository: aesop/aesop-github-action
+          token: ${{ secrets.X_GITHUB_TOKEN }}
+          path: actions
+
+      - name: Update Gel
+        uses: ./actions/create-pr
```
Comment (kevin-ho87) [inline .github/workflows/tag-push.yml:40]: How come this path uses `./actions/..` and the others use `actions/...`.
Are they just in different path locations?

Comment (MkMan) [reply .github/workflows/tag-push.yml:40]: Ah 😃 
* `actions/...` refers to actions from the GH marketplace. For example, [actions/checkout](actions/checkout@v2)
* `./actions/create-pr` is referring to a [custom action](https://github.com/aesop/aesop-github-action/tree/master/create-pr) created by us (Aesop) and cloned in the folder `actions`. This is done in the previous step

Comment (kevin-ho87) [reply .github/workflows/tag-push.yml:40]: 👍  makes sense

**Comment by beau-charman** [comment]
How is this one going @MkMan ?

**Comment by MkMan** [comment]
> How is this one going @MkMan ?

Just waiting for an approval to merge it 👍

**Review by maxlew — APPROVED** [review]
Sorry let this one sit for a while, looks good though 👍🏻

**Pull request merged**